cmd/federation-controller/main.go

```diff
@@ -45,6 +35,13 @@ import (
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds"
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds/adsc"
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds/adss"
+	routev1client "github.com/openshift/client-go/route/clientset/versioned"
+	istiokube "istio.io/istio/pkg/kube"
+	istiolog "istio.io/istio/pkg/log"
+	"istio.io/istio/pkg/slices"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/rest"
```

Comment:
It seems that goimports does not work as expected...

Reply:
That's Goland sorting it alphabetically. I think that's better this way. I will eventually automate it, but no time now :)